modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/Settings.java

```diff
@@ -29,6 +29,9 @@ public class Settings implements Runnable, ISettings {
     public static final int MAX_TEST_POD_RETRY_LIMIT_DEFAULT = 5;
     public static final String MAX_TEST_POD_RETRY_LIMIT_CONFIG_MAP_PROPERTY_NAME = "max_test_pod_retry_limit";
 
+    public static final int TEST_POD_INTERRUPT_TIMEOUT_SECS_DEFAULT = 300;
+    public static final String TEST_POD_INTERRUPT_TIMEOUT_SECS_PROPERTY_NAME = "test_pod_interrupt_timeout_secs";
```

Comment:
testInterruptCleanupGracePeriod ?

Reply:
I think `interruptedTestRunCleanupGracePeriodSeconds` might be easier to read, or perhaps just `testRunCleanupGracePeriodSeconds`?

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/376/commits/18fe87da32d39dda21609f1eb11d8343a304fbfa